jetbrains-core/src/software/aws/toolkits/jetbrains/services/clouddebug/actions/StartRemoteShellAction.kt

```diff
@@ -120,25 +116,21 @@ class StartRemoteShellAction(private val project: Project, private val container
                 }
 
                 override fun onSuccess() {
-                    recordTelemetry(TelemetryConstants.TelemetryResult.Succeeded.name)
+                    recordTelemetry(Result.SUCCEEDED)
                 }
 
                 override fun onThrowable(error: Throwable) {
-                    recordTelemetry(TelemetryConstants.TelemetryResult.Failed.name)
+                    recordTelemetry(Result.FAILED)
                 }
 
-                private fun recordTelemetry(result: String) {
-                    TelemetryService.getInstance().record(project) {
-                        datum("${CLOUDDEBUG_TELEMETRY_PREFIX}_shell") {
-                            createTime(startTime)
-                            (cloudDebugExecutable as? ExecutableInstance.Executable)?.version?.let {
-                                metadata(CLOUDDEBUG_VERSION, it)
-                            }
-                            metadata(RESULT, result)
-                            unit(Unit.MILLISECONDS)
-                            value(Duration.between(startTime, Instant.now()).toMillis().toDouble())
-                        }
-                    }
+                private fun recordTelemetry(result: Result) {
+                    ClouddebugTelemetry.shell(
+                        project,
+                        version = (cloudDebugExecutable as? ExecutableInstance.Executable)?.version,
```

Comment:
Can be cleaned u after you merge SAM changes

Reply:
yep, will add comment